packages/mds-db/types.ts

```diff
@@ -78,6 +78,7 @@ export interface VehicleEventCountResult {
 
 export interface ReadGeographiesParams {
   get_published: Nullable<boolean>
+  only_published_after?: Timestamp
   get_unpublished: Nullable<boolean>
   geography_ids?: UUID[]
```

Comment:
I like this conceptually, but this is starting to look a bit like parameter plinko. I wonder at what point having separate `getGeographies` and `getPublishedGeographies` and the like makes sense.

Reply:
Simply because `getGeographies({ get_published: false, only_published_after: now(), get_ubpublished: true })` and other combinations don't make a whole lot of sense.

I think a set of methods is more intuitive
```ts
// Get all or by ID
getGeographies({ geography_ids?: UUID[] })
// Get all unpublished
getUnpublishedGeographies()
// Get all published (with optional cursor)
getPublishedGeographies({ only_published_after? Timestamp })
```

Reply:
yeah, definitely. Thanks!